index.js

```diff
@@ -188,8 +202,6 @@ VirtualModulesPlugin.prototype.apply = function(compiler) {
         this._virtualFiles[file] = {stats: stats, contents: contents};
         setData(statStorage, file, createWebpackData(stats));
         setData(fileStorage, file, createWebpackData(contents));
-        self._compiler.fileTimestamps instanceof Map &&
-          self._compiler.fileTimestamps.set(file, +stats.mtime);
```

Comment:
Wouldn't removing this cause issues for webpack 4?

Reply:
`fileWatcher.directoryWatcher.setFileTime` call added in this PR should work both for Webpack 4 and Webpack 5, at least, when I checked it worked for me, we lack proper automated tests now and they are hard to write, so yeah, if you noticed that this change breaks something, please open an issue.